.gitattributes

```diff
@@ -1 +1 @@
-*.* text eol=lf
\ No newline at end of file
+*.* text=auto eol=lf
```

Comment:
What does this do?

Reply:
this is a cherry pick from 3b1a2709abc11f11d4822c9c3b936c7a0f7e1492

It prevents git from trying to convert LFs on non-text files.

Reply:
main already has this change though

Reply:
Got it, was just curious

Reply:
I cherry picked it to all active branches to try and prevent gradle from being broken.